documentation/howto/import_FH_app.adoc

```diff
@@ -0,0 +1,189 @@
+= Import Your FeedHenry Application into the IDE
+:page-layout: howto
+:page-tab: docs    
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+Use Red Hat JBoss Developer Studio’s `Import` wizard to import a FeedHenry application into your workspace. When imported, edit the application as required and push it back to the FeedHenry server. 
+
+Follow the provided steps to import and make changes to a FeedHenry application using your Red Hat JBoss Developer Studio workspace:
+
+. <<import_app, Import Your FeedHenry Application>>
+. <<change_app, Change the Application>>
+. <<test_app, Test the Application>>
+. <<push_changes_to_FH, Push the Changes Back to the FeedHenry Server>>
+
+== Prerequisites 
+Ensure that the following prerequisites are met to enable the FeedHenry feature in JBoss Developer Studio:
```

Comment:
again - this content is for tools.jboss.org so I suggest we keep references to JBoss Developer Studio at a minimum.